doc/setup.md

```diff
@@ -181,20 +181,79 @@ To get started using the application with docker,
 1. Install [Docker](https://www.docker.com/get-started)
 2. Install [docker-compose](https://docs.docker.com/compose/install/)
 3. Clone the repository, and open the repository folder in your favorite command line or terminal application.
-4. From within the repository, navigate to the `/docker/development` folder. If you are in the right folder, you will see a file named `docker-compose.yml` and a file named `Makefile`.
-5. Now you should be able to run `make start`. This will start the application in daemon mode, which means that the server will keep running in the background. If you navigate to  `localhost:3000` in your browser, you will see an error. This is normal, and it means that you still need to setup the database.
-6. To setup the database, you can run
-  `SYSTEM_EMAIL=${SYSTEM_EMAIL} SYSTEM_PASSWORD=${SYSTEM_PASSWORD} make seed`
-  This will setup the database and create a default admin user with the email and password as specified by the `SYSTEM_EMAIL` and `SYSTEM_PASSWORD` environment variables which are ultimately passed to `docker-compose` with the `-e` option. If you don't want to create the default user, you can just run `make prepare` and create the account using the sign up option on the website.
 
-7. You should now be able to reload `localhost:3000` in your browser. If everything went well, the website should appear and be functional. You can sign in using the email and password you set in the previous step. This docker compose also setups an a `mailcatcher` server, which you can access at `localhost:1080`. All emails will be delivered to mailcatcher, which should allow you to setup user accounts.
+### Bootstrapping the application for development
 
-**NOTE** Do not use this method in production! This is for **testing & development only** the configuration used with in this docker-compose file is highly insecure and should never be exposed to the public internet.
+```
+bin/bootstrap
+```
+
+Tears down any exiting images, containers, networks, etc. and sets everything up from scratch. This also seeds the development database - you will be prompted to enter your desired credentials for the development admin user.
+
+### Running tests
+
+```
+bin/test-docker
```

Comment:
I'm not in love with the name of this particular script, but `bin/test` was already present and runs both the RSpec suite and Mocha JS suite on the host. This felt like an adequate compromise, but I'd love to rename it if anyone has a better naming suggestion.

Reply:
On the topic of naming, i'm wondering whether it'd be worth separating the docker binstubs from the rest?
Instead of `bin/`, they could live in (brainstorming...):
```
bin/dc/
bindc/
dcbin/
dc/
dev/
?/
```
This would cleanly separate host binstubs from those that trigger `docker-compose` and also allow us to reuse names such as `test` w/o conflict.

Reply:
What do you think about a single, contained, self-documenting CLI tool for managing the docker development environemnt - maybe a small [thor](https://github.com/erikhuda/thor/wiki) app? We could then move all of this disparate binstubs into something like a single binstub `bin/dev` and provide an interface like:

```
bin/dev help # Explain what `bin/dev` is and show-off what it can do
bin/dev bootstrap # current functionality of `bin/bootstrap`
bin/dev start # current functionality of `bin/start`
bin/dev test # current functionality of `bin/test-docker
```

and so on.

Reply:
I'm going to move forward with moving all of the existing scripts to from `bin/` to `bin/dev/` as an interim step. Thinking about the CLI tool a bit more, it seems like overkill right now - moving the scripts to `bin/dev/` feels like it has ship-able value.

Additionally, there are a few chicken egg issues to work out. For example, the above mentioned approach of a `thor` app would require ruby and thor to be present on the host development machine - kinda defeats the purpose of using docker for development. Running the thor application inside the container also poses some issues as most of the "commands" are to control docker services on the host machine. Seems like this is a good state to put this down for a bit and put in some deep thunks on future incremental improvements while making progress on some other objectives.

Reply:
interesting!

Reply:
Addressed by: https://github.com/rubyforgood/mutual-aid/pull/715/commits/11597893b4a12d79a77d611d47aa9135d26e1e47